examples/js/effects/VREffect.js

```diff
@@ -233,6 +233,20 @@ THREE.VREffect = function ( renderer, onError ) {
 
 	};
 
+	this.requestAnimationFrame = function ( f ) {
+
+		if ( ! isDeprecatedAPI && vrHMD !== undefined ) {
```

Comment:
I think this:

```javascript
var rAFHandle;
this.requestAnimationFrame = function ( f ) {
  if ( ! isDeprecatedAPI && vrHMD !== undefined ) {
    rAFHandle = vrHMD.requestAnimationFrame( f );
  } else {
    rAFHandle = window.requestAnimationFrame( f );
  }
};

this.cancelAnimationFrame = function () {
  if ( ! isDeprecatedAPI && vrHMD !== undefined ) {
    vrHMD.cancelAnimationFrame( rAFHandle );
  } else {
    window.requestAnimationFrame( rAFHandle );
  }
};
```

But maybe I don't see many people using so may not need to add. If they need they can getVRDisplay (when add) and handle themselves. auto submitFrame flag makes sense though like in other PR landing. 

I kind of think the VRDisplay should be an optional parameter to VREffect (and VRControls) so can handle on own & have a reference to, would support multiple headsets at a time (or just choosing), etc.